benchmark/ext_helpers/daal4py.py

```diff
@@ -21,45 +20,63 @@
 else:
     from daal4py.sklearn.cluster._k_means_0_23 import KMeans
 
-
 from sklearn.exceptions import NotSupportedByEngineError
+from sklearn_numba_dpex.kmeans.engine import KMeansEngine
+
+
+# TODO: instead of relying on monkey patching the defau;t engine, find a way to
+# register a distinct entry point that can load a distinct engine outside of setup.py
+# (impossible ?)
```

Comment:
Not sure it's worth it for now. We will reconsider this once we have implemented several engine entrypoints in wip-engines.

Reply:
Agree, however it's also an important weakness of the current entry point approach, with the current state of our plugin system users can't just write a new engine in a jupyter notebook and load it into the plugin api, it seems that entry points are _required_ to be declared in the setup file. Maybe we could prioritize investigating this point more, because it might be a deterrent for using `entry_points` at all.